app/views/registrations/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <% provide(:title, I18n.t('registrations.list.title', comp: @competition.name)) %>
 
 <%= render layout: "nav" do %>
-  <%= react_component('RegistrationsV2/Registrations', { competitionInfo: @competition.to_competition_info }) %>
+  <%= react_component('RegistrationsV2/Registrations', { competitionInfo: @competition.to_competition_info, userInfo: @current_user }) %>
```

Comment:
Fully serializing a user is usually expensive, because the resulting JSON contains a _lot_ of information by default. Depending on what information you need, you can do:

```suggestion
  <%= react_component('RegistrationsV2/Registrations', { competitionInfo: @competition.to_competition_info, userInfo: @current_user.as_json(
    only: [...names of database columns...],
    methods: [...names of methods...],
    include: [...ORM relations to include, like current teams for example...]
  ) }) %>
```